ecs-logs-collector.sh

```diff
@@ -584,6 +585,24 @@ enable_ecs_agent_debug() {
   esac
 }
 
+get_docker_goroutines() {
+  try "dump docker goroutine stacks"
+  dstdir="${info_system}/docker_stacks"
+  mkdir -p ${dstdir}
+  # send SIGUSR1 to dump goroutines to
+  # /var/run/docker/goroutine-stacks-<date>.log
+  if kill -SIGUSR1 $(pidof dockerd) ; then
+    # give docker time to write the file
+    sleep 5
```

Comment:
Instead of waiting for `$hopefully_long_enough`, can we instead watch for the expected files to be created before continuing?

Reply:
What do you think is the best way to do that?

Reply:
Clearly lost track of this but would doing 

```bash
until find /var/run/docker/ -cmin -1 -type f -name "goroutine-stacks*.log" -exec cp -p '{}' "$dstdir" \; 
    sleep 1;
done
```
Be acceptable?

Reply:
That looks like that'd catch the right set of files. Did you have a few successful runs with this? Did it catch all the goroutine stack dumps? I think this method will be robust enough for most cases. If it turns out that timing is even more finicky then I'd say let's spend more time on this. Until then, let's move forward with this.